data/language/english_uk.txt

```diff
@@ -4094,6 +4094,7 @@ STR_5786    :Invalid group name
 STR_5787    :{COMMA32} players online
 STR_5788    :Default inspection interval:
 STR_5789    :Disable lightning effect
+STR_5790    :Disable train length limit
```

Comment:
This string now collides with #3219 & later on with PR #3241 
Should be changed to STR_5803